src/app/games/coerceo/coerceo.component.ts

```diff
@@ -84,12 +84,14 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         if (move instanceof CoerceoRegularMove) {
             this.lastStart = MGPOptional.of(move.getStart());
             this.lastEnd = MGPOptional.of(move.getEnd());
-        } else {
-            this.lastStart = MGPOptional.empty();
-            this.lastEnd = MGPOptional.empty();
         }
     }
 
+    public override hideLastMove(): void {
```

Comment:
problème de viewbox lié au triangle, arrivé dans cette branche !
https://lifecycle-refactor-slm-and-hlm--everyboard.netlify.app/local/Coerceo